javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var state = false;
```

Comment:
Why don't we rename this to something that better conveys its purpose. Maybe `isLightbulbOn`